### PR TITLE
Bug 1242396 - [TV][Metrics] Track "Send to TV" usage

### PR DESCRIPTION
**Pull request by rexboy7**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c153fe75db925b6ad8bc3be0ebba809e91d38af0

File: tv_apps/smart-system/js/app_usage_metrics.js [hunk 1]
```diff
@@ -454,6 +458,17 @@
       this.currentApp = e.detail;
       this.currentAppStartTime = now;
       break;
+    case APPLOADED:
+      // For "send to TV" feature only.
+      // For now, notification-receiver launches every time it receive send to
+      // TV request. So invocation count of notification-receiver should be
+      // equal to the count of send to TV usage count.
```
Comment (luke-chang) [inline tv_apps/smart-system/js/app_usage_metrics.js]: Please also mention that "notification-reveiver" will never become foreground so we can't track it by "appopened".

File: tv_apps/smart-system/js/app_usage_metrics.js [hunk 2]
```diff
@@ -454,6 +458,17 @@
...
+      var loadedApp = e.detail;
+      if (loadedApp.manifestURL &&
+          loadedApp.manifestURL.indexOf(SENDTOTV_ORIGIN) === 0) {
```
Comment (luke-chang) [inline tv_apps/smart-system/js/app_usage_metrics.js]: nit: You can use `startsWith` here but it's up to you.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=453999d04b5d882cd62fe718072b8ba0a76c428f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5a344f758beba007e580a6e088ee55b4ff43a296

**Pull request merged**